fenix/app/src/main/res/layout/preference_widget_radiobutton_with_info.xml

```diff
@@ -10,8 +10,7 @@
     android:layout_height="wrap_content"
     android:baselineAligned="false"
     android:orientation="horizontal"
-    android:paddingTop="@dimen/radio_button_preference_vertical"
-    android:paddingBottom="@dimen/radio_button_preference_vertical">
+    android:paddingBottom="12dp">
```

Comment:
This file, given [the design](https://www.figma.com/file/OUOpOzgd4EOdgwJZvBGQ9j/TCP?node-id=248-13737&t=SGvuya4sjmx9Mvau-4), needs some adjustments.

Top spacing from the title should be `6dp`, bottom spacing from the `widget_summary` should be `6dp` as well. The info icon should be positioned in center (currently bottom padding of the parent view is lifting it up a bit).

Reply:
Thanks!

Reply:
Good spot, a rogue top margin on title. Updated per screenshot
<img width="396" alt="Screenshot 2023-03-29 at 11 40 26" src="https://user-images.githubusercontent.com/1833156/228509181-5772b2bb-d0a4-4fb4-97e8-7f98e6ff8250.png">